tests/Tokenizer/TokenTest.php

```diff
@@ -285,24 +285,34 @@ public function provideIsWhitespaceCases()
         );
     }
 
-    public function testPropertiesOfArrayToken()
+    /**
+     * @param array|string|mixed $prototype
```

Comment:
`array|string|mixed` sounds like _"can be anything, or an array or a string"_ which feels weird and is still an invalid subset.